Shared/Samples/Add building scene layer/AddBuildingSceneLayerView.swift

```diff
@@ -0,0 +1,106 @@
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct AddBuildingSceneLayerView: View {
+    /// A local scene with topographic basemap style and a tiled elevation source.
+    @State private var scene: ArcGIS.Scene = {
+        let scene = Scene(
+            basemapStyle: .arcGISTopographic,
+            viewingMode: .local
+        )
+        
+        // Adds a surface.
+        
+        let elevationSource = ArcGISTiledElevationSource(
+            url: URL(string: "https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer")!
+        )
+        scene.baseSurface.addElevationSource(elevationSource)
+        
+        return scene
+    }()
+    
+    /// A Boolean value that indicates if the full model of the building scene layer is showing or not.
+    @State private var showsFullModel = false
+    
+    /// The camera for zooming the local scene view to the building scene layer.
+    @State private var camera: Camera? = Camera(
+        location: Point(x: -13_045_109, y: 4_036_614, z: 511, spatialReference: .webMercator),
+        heading: 343,
+        pitch: 64,
+        roll: 0
+    )
+    
+    /// The overview sublayer which represents the exterior shell of the building.
+    @State private var overviewSublayer: BuildingSublayer?
+    
+    /// The full model sublayer which contains all the features of the building.
+    @State private var fullModelSublayer: BuildingSublayer?
+    
+    var body: some View {
+        LocalSceneView(scene: scene, camera: $camera)
+            .task {
+                let buildingSceneLayer = BuildingSceneLayer(
+                    url: URL(string: "https://3dcities.maps.arcgis.com/home/item.html?id=ebae5f766aac41ba9ec588f66028a6a9")!
+                )
+                
+                // Sets the altitude offset of the building scene layer.
+                // Upon first inspection of the model, it does not line up with the global
+                // elevation layer perfectly. To fix this, add an altitude offset to align
+                // the model with the ground surface.
+                buildingSceneLayer.altitudeOffset = 1
+                
+                try? await buildingSceneLayer.load()
+                
+                // Adds building scene layer to scene.
+                scene.addOperationalLayer(buildingSceneLayer)
+                
+                // Get the overview and full model sublayers for the
+                // toggle.
```

Comment:
```suggestion
                // Gets the overview and full model sublayers for the toggle.
```

<img width="952" height="288" alt="Image" src="https://github.com/user-attachments/assets/f107cd5c-3b8f-4d9e-91eb-732e511d9ef0" />

Seems the column guide is inconsistent…